src/BaseGL.js

```diff
@@ -321,10 +344,124 @@ class BaseGL {
       e.preventDefault();
 
       const hdata = e.detail.data;
+
+      // Only run this code if hi
+      if (hdata && hdata.indices.length > 0 && this.nrows) {
+        const index = hdata.indices[0]; // handle only one point
+        const col = Math.floor(index / this.nrows);
+        const row = index % this.nrows;
+
+        // Invert row, considering X axis starts from bottom up
```

Comment:
Is this fixed or a setting? If a setting, we should make sure the current plot is using an inverted axis.

Reply:
I fixed it temporarily, I have to discuss it with jay about how to determine whether the axis is going to be inverted from the input. I couldn't find any clue. But will sure look into it before merging.

Reply:
currently all visualizations share the same axis center (in epiviz.gl). This might be a bit confusing for heatmaps, but if you prefer to invert the order, I recommend we do this in the heatmap library and not the gl library